src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -967,6 +973,19 @@ private void WriteApphostsForShimRuntimeIdentifiers()
                 }
             }
 
+            /// <summary>
+            /// After netcoreapp3.0 apphost is resolved during ResolveFrameworkReferences. It should return nothing here
+            /// </summary>
+            private static bool CannotResolveApphostFromAssetFile(NuGetFramework targetFramework)
```

Comment:
Nit: using a negative assertion in a method name can be confusing.  Perhaps call this `CanResolveApphostFromFrameworkReference` or `CanResolveApphostFromAssetsFile` and use `if (!CanResolveApphostFromAssetsFile(...)))`?

Reply:
Was this feedback going to be resolved as part of this PR?

Reply:
Sorry i miss this. I talked to Daniel, we still need this check in since assets will not be in asset.json for 3.0

Reply:
👍 what about Daniel's comment regarding checking the target framework in addition to the version?

Reply:
ah, let me add tfm check